### PR TITLE
VIITE-1862 create calculated nodepoints with node and project change

### PR DESCRIPTION
**Pull request by peltsi212**

### CODE REVIEW

**Comment by peltsi212** [comment]
Jenkins test is ok.

File: .gitignore [hunk 1]
```diff
@@ -13,3 +13,8 @@ project/project/
 yarn.lock
 package-lock.json
 digiroad2.boot.log
+/build/
+.classpath
+.project
+.settings/
+WebContent/
```
Comment (andre-policarpo) [inline .gitignore]: Is this suppose to be merged? Just checking.

Comment (peltsi212) [reply .gitignore]: Yes these changes enable to use eclipse. I use eclipse for git stashes

Comment (andre-policarpo) [reply .gitignore]: Nothing against Eclipse, it was my first ever IDE, but you should really give it a try to IntelliJ IDE! 😄

File: digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/dao/NodeDAO.scala [hunk 2]
```diff
@@ -337,4 +337,27 @@ class NodeDAO extends BaseDAO {
       queryList(query)
     }
   }
+
+  def fetchNodeNumbersByProject(projectId: Long): Seq[Long] = {
```
Comment (skosonen) [inline digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/dao/NodeDAO.scala]: Should we join here also the node -table and check that the node.end_date is null and node.valid_to is null.

**Comment by skosonen** [comment]
Tests pass on Jenkins.

**Comment by skosonen** [comment]
All tests are still passing.

**Pull request merged**